### PR TITLE
Add package for Screencloud 1.2.0

### PR DESCRIPTION
**Pull request by Forkk**
Added a package for Screencloud, a program for taking screenshots and
uploading them to various websites.

This package was a bit finicky due to some nonsense in the CMakeLists
file, but it seems to work properly.

The package also includes API keys for the screencloud.net website.
According to the author of the Arch Linux User Repo package for
screencloud, these keys are only used for usage tracking, so we should
be fine including them in the package here.

I've also added a package for Qxt, and fixed the QtMobility package.

### CODE REVIEW

File: pkgs/development/libraries/qxt/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  patchPhase = ''
+    patchShebangs configure
+    substituteInPlace configure --replace "/bin/pwd" "${coreutils}/bin/pwd"
+  '';
+
+  prefixKey = "-prefix ";
+
+  meta = {
+    homepage = http://libqxt.org;
+    description = "An extension library for Qt providing a suite of cross-platform utility classes to add functionality not readily available in the Qt toolkit by Qt Development Frameworks, Nokia.";
```
Comment (jagajaga) [inline pkgs/development/libraries/qxt/default.nix]: Can you divide this big line into several lines? Like here https://github.com/NixOS/nixpkgs/pull/7389/files#diff-a8578a6cebc31660fb8c05a29f6b6306R18
And also put it into `longDescription` section.

Comment (Forkk) [inline pkgs/development/libraries/qxt/default.nix]: Sounds fine to me. Will do it when I get a chance (working on a different package right now), but what about the short description?

Comment (jagajaga) [inline pkgs/development/libraries/qxt/default.nix]: Oh, yeah. Also add `description`. Make it short :)

Comment (Forkk) [inline pkgs/development/libraries/qxt/default.nix]: Would "An extension library for Qt." do, or is that too short?

File: pkgs/development/libraries/qxt/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  patchPhase = ''
+    patchShebangs configure
+    substituteInPlace configure --replace "/bin/pwd" "${coreutils}/bin/pwd"
+  '';
+
+  prefixKey = "-prefix ";
+
+  meta = {
+    homepage = http://libqxt.org;
+    description = "An extension library for Qt.";
```
Comment (jagajaga) [inline pkgs/development/libraries/qxt/default.nix]: It's ok, but please remove dot at the end

**Comment by jagajaga** [comment]
And don't create additional commits for changes. Squash into existing one for that file.

**Comment by Forkk** [comment]
Alright, squashed commits and removed the dot.

**Comment by jagajaga** [comment]
Awesome! Thank you.

**Pull request merged**